build/ci/downloadRenderer.js

```diff
@@ -63,6 +71,20 @@ async function downloadRendererExtension() {
             path.join(ExtensionRootDir, 'out', 'client_renderer')
         );
         console.log('Copied Renderer extension output.');
+
+        // Download and extract ipywidget renderer js
+        const downloadedWidgetFile = path.join(downloadDir, widgetFileName);
```

Comment:
This is the renderer JS file that I uploaded to blob storage.

Reply:
Right now the project is just local to my machine. I'll get it up somewhere for review. But it was basically just using the vscode yeoman code generator to build a renderer extension and then swapping in just our single renderer file and updating to the new API. I've tested basic widgets with Jupyter downloading this file and they work.